nix/tests/expected/auth.out

```diff
@@ -44,6 +44,138 @@ order by
  auth        | users             | supabase_auth_admin | f           | 
 (5 rows)
 
+-- auth schema objects with roles privileges
+select
+  ns.nspname    as schema_name,
+  c.relname     as table_name,
+  r.rolname     as role_name,
+  a.privilege_type,
+  a.is_grantable
+from
+  pg_class      c
+join
+  pg_namespace  ns  on c.relnamespace = ns.oid
+cross join lateral
+  aclexplode(c.relacl) as a
+join
+  pg_roles      r   on a.grantee = r.oid
+where
+  ns.nspname = 'auth'
+  and c.relkind in ('r', 'v', 'm')
+  and a.privilege_type <> 'MAINTAIN'
+order by
+  c.relname,
+  r.rolname,
+  a.privilege_type;
+ schema_name |    table_name     |      role_name      | privilege_type | is_grantable 
+-------------+-------------------+---------------------+----------------+--------------
+ auth        | audit_log_entries | dashboard_user      | DELETE         | f
+ auth        | audit_log_entries | dashboard_user      | INSERT         | f
+ auth        | audit_log_entries | dashboard_user      | REFERENCES     | f
+ auth        | audit_log_entries | dashboard_user      | SELECT         | f
+ auth        | audit_log_entries | dashboard_user      | TRIGGER        | f
+ auth        | audit_log_entries | dashboard_user      | TRUNCATE       | f
+ auth        | audit_log_entries | dashboard_user      | UPDATE         | f
```

Comment:
Note: these privileges could be compressed to a single line with `array_agg`, but then once a test fails is hard to visualize the privileges diff. (case in point: just happened to me with the new MAINTAIN privilege on pg 17, the compressed privs made that hard to visualize).